layouts/shortcodes/cve-feed.html

```diff
@@ -1,9 +1,19 @@
 {{ $feed := getJSON .Site.Params.cveFeedBucket }}
```

Comment:
Really, we should switch this to use `transform.GetRemote`; see https://gohugo.io/functions/resources/getremote/#remote-data for an example that includes error handling. Then I don't think we need to edit the `hugo.toml` to ignore `error-remote-getjson`.

(If we switch, we can still change the error handling based on which environment we're in)